src/Test.FunctionalTests.Sarif/SarifValidatorTests.cs

```diff
@@ -197,7 +197,7 @@ public void ValidatesAllTestFiles()
             Path.Combine("v2", "ConverterTestData"),
             Path.Combine("v2", "SpecExamples"),
             Path.Combine("v2", "ObsoleteFormats"),
-            Path.Combine("..", "..", "Test.UnitTests.Sarif", "netcoreapp3.1", "TestData")
+            Path.Combine("..", "..", "Test.UnitTests.Sarif", "net6.0", "TestData")
```

Comment:
>net6.0 [](http://example.com/codeflow?start=62&length=6)

fyi Suvam mentioned to me coyote testing does not support 6.0 yet, see if we want to keep in 3.1